src/signup/signup.js

```diff
@@ -9,16 +9,57 @@ import CssBaseline from "@material-ui/core/CssBaseline";
 import Typography from "@material-ui/core/Typography";
 import Button from "@material-ui/core/Button";
 import styles from "./styles";
-import { Form } from "reactstrap";
+
 const firebase = require("firebase");
```

Comment:
You have a file at `src/firebase.js`. Inside this file is where you initialise firebase, and export it for use in the app.

So here in `signup.js` you should be importing `src/firebase`, not the firebase package itself. So I think this line should be replaced with 
```
import firebase from '../firebase'
```

Reply:
Ok, I replaced the line with `import firebase from '../firebase'`